src/Microsoft.DotNet.VersionTools/Automation/VersionsRepoUpdater.cs

```diff
@@ -188,6 +197,12 @@ public VersionsRepoUpdater(GitHubAuth gitHubAuth, GitHubProject project)
                 path,
                 new GitHubBranch("master", _project));
 
+            if (latestPackages == null)
+            {
+                Trace.TraceInformation($"No versions repository file '{path}' found.");
```

Comment:
This one's redundant--I guess I was thinking that it would be good to have this info if we use `GetPackagesAsync` again for something else and don't log the null case at that call site. I don't think that's likely, so I'll remove it.